docroot/themes/humsci/su_humsci_theme/su_humsci_theme.libraries.yml

```diff
@@ -94,6 +94,13 @@ base:
     base:
       css/base/base.css: {}
       css/components/molecules/hover-menu.css: {}
+
+archaeology:
+  version: VERSION
+  css:
+    base:
```

Comment:
Is this really base css? Maybe right now it is but I suspect you have had to make your selector stronger because it is losing out to other higher level css. I suspect you could move this to theme or layout and get ride of the #main-content wrapper on your heading styles.

From D.o
```
base: CSS reset/normalize plus HTML element styling. Key assigns a weight of  CSS_BASE = -200

layout: macro arrangement of a web page, including any grid systems. Key assigns a weight of CSS_LAYOUT = -100

component: discrete, reusable UI elements. Key assigns a weight of CSS_COMPONENT = 0

state: styles that deal with client-side changes to components. Key assigns a weight of CSS_STATE = 100

theme: purely visual styling (“look-and-feel”) for a component. Key assigns a weight of CSS_THEME = 200
```

Reply:
yeah it might be more `theme` level? since this would be final changes to subtheme stuff

Reply:
Good catch! You're right, and thanks for citing Drupal's definition here. I'll update to `theme:`.

I will also remove the `#main-content`. We will want to be sure that we remove the dash from any unintended `<h*>` tags that inherit the style. But, better to be broad and get more narrow as needed that the other way around (at least in this case).